consensus/src/util/time_service.rs

```diff
@@ -120,8 +122,8 @@ impl TimeService for ClockTimeService {
         aptos_infallible::duration_since_epoch()
     }
 
-    fn sleep(&self, t: Duration) {
```

Comment:
Was this causing some slowness and thread contention during network timeouts?

Reply:
not sure, the sleep only happens when the block timestamp is ahead of local timestamp which should be rare, we also have a counter for that